### PR TITLE
add launch info

### PR DESCRIPTION
**Pull request by Juphex**
adjust readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ To build the full software stack one needs to execute the following command in t
 cd catkin_ws
 
 catkin build
+# alternatively: catkin_make
```
Comment (Flova) [inline README.md:89]: I would strongly discourage people from building with catkin_make. I would not mention it.

File: README.md [hunk 2]
```diff
@@ -99,7 +100,12 @@ cd catkin_ws
 source devel/setup.bash
 ```
 
-Now you can launch the software. 
+Now you can launch the bringup package that brings up the launch file for each package:
+
+```bash
+roslaunch marimbabot_bringup marimbabot.launch
+```
+
```
Comment (Flova) [inline README.md:108]: Maybe also add the vision launch file

Comment (Juphex) [reply README.md:108]: Do you want the marimbabot_bringup launch file to include all other .launch files?
Or what is the purpose of bringup in your opinion?

File: README.md (hunk 1 above)
Comment (Juphex) [reply README.md:89]: I build with catkin_make only. But I see your point - it seems more robust to changes etc..

File: README.md (hunk 2 above)
Comment (Flova) [reply README.md:107]: Normally each package e.g. vision have their own launch files in their "launch" folder.
The bringup launches all components and gets the full system up and running.

File: README.md [hunk 3]
```diff
@@ -99,7 +99,15 @@ cd catkin_ws
 source devel/setup.bash
 ```
 
-Now you can launch the software. 
+Now you can launch the bringup package that brings up the launch file for each package:
+
+```bash
+roslaunch marimbabot_bringup marimbabot.launch
+
+# or launch a single package
```
Comment (Flova) [inline README.md:107]: I would consider adding a new subsection for the subcomponents instead of adding a comment.

Comment (Juphex) [reply README.md]: k

**Review by Flova — APPROVED** [review]

**Pull request merged**